modules/space/rendering/renderablerings.cpp

```diff
@@ -73,11 +73,11 @@ namespace {
         "of the night side occurs."
     };
 
-    constexpr openspace::properties::Property::PropertyInfo TransparencyInfo = {
-        "Transparency",
-        "Transparency",
-        "This value determines the transparency of part of the rings depending on the "
-        "color values. For this value v, the transparency is equal to length(color) / v."
+    constexpr openspace::properties::Property::PropertyInfo TranslucencyInfo = {
```

Comment:
Personally I think "ColorThreshold" or "ColorFilter"  is more descriptive to what this property is in charge of. Maybe @jccosta has a good explanation of its purpose.

Reply:
I don't have a good explanation for that name, @liuloppan. It was there already when I had to make changes related to it. :smile: 
This renderable is not currently in use in OS. We are using the https://github.com/OpenSpace/OpenSpace/blob/master/modules/globebrowsing/src/ringscomponent.cpp for rendering rings with shadows. 
The only reason the renderable was not deleted is that someone may have a use for a standalone ring (no planet) or something else.

Reply:
@liuloppan and @jccosta, I made the same change in `ringscomponent`, so whatever name we come up with should be changed in that one as well :) I do like the "ColorFilter" suggestion

Reply:
I am okay with either.

Reply:
Color filter or keeping it Translucency sound good to me